requirements/strategies.txt

```diff
@@ -0,0 +1,3 @@
+fairscale>=0.4.5
+deepspeed>=0.6.0
```

Comment:
Regarding the deepspeed version `deepspeed>=0.6.0`,

- isn't this too strict? It should support `<0.6.0` as well, no? cc @SeanNaren
- 0.6.0 and 0.6.1 are incompatible due to https://github.com/microsoft/DeepSpeed/issues/1878.
- 0.6.2 is incompatible due to its installation issue https://github.com/microsoft/DeepSpeed/issues/1918.